006-vortexa-tech-challenge/src/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { ThemeProvider, createTheme } from "@mui/material/styles";
+
+import { App } from "./app";
+import CssBaseline from "@mui/material/CssBaseline";
+import React from "react";
+import ReactDOM from "react-dom";
```

Comment:
Similarly here, sort the imports.

```suggestion
import React from "react";
import ReactDOM from "react-dom";
import { ThemeProvider, createTheme } from "@mui/material/styles";
import CssBaseline from "@mui/material/CssBaseline";

import { App } from "./app";
```